docs/src/main/paradox/java/http/common/http-model.md

```diff
@@ -346,6 +346,14 @@ Similarily to the media types registration, you can register custom status codes
 
 @@snip [CustomStatusCodesExampleTest.java](../../../../../test/java/docs/http/javadsl/CustomStatusCodesExampleTest.java) { #application-custom-java }
 
+<a id="registeringcustommethod-java"></a>
+## Registering Custom HTTP Method
+
+Akka HTTP also allows you to define custome HTTP methods, other than the well-known methods @javadoc[predefined](akka.http.javadsl.model.HttpMethods) in Akka HTTP.
+To use a custom HTTP method, you need to define it, and then add it to parser settings like below:
```

Comment:
Nitpicking here, but maybe say "define it first, and then..."
`To use a custom HTTP method, you need to define it first, and then add it to parser settings like below:`